start-fluentd

```diff
@@ -32,16 +34,17 @@ cat << 'EOF' > /etc/fluent/fluent.conf
 
 <filter kubernetes.**>
   type kubernetes_metadata
+  preserve_json_log ${KUBERNETES_PRESERVE_JSON_LOG}
```

Comment:
Oops just noticed this needs moving in to the other heredoc - fixing up in separate commit.